### PR TITLE
Kirkstone move base files

### PR DESCRIPTION
**Pull request by mikehaller**
base-files and systemd-conf were in incorrect locations in meta-leda. Due to a misconfiguration in layer.conf of meta-leda-distro, the recipes were still being found in the unusual place.

After this pull request, base-files and systemd-conf recipes and extra files are in the expected locations, where a normal layer configuration would look for the files.

### CODE REVIEW

File: meta-leda-bsp/conf/machine/qemux86-64-extra.conf [hunk 1]
```diff
@@ -10,3 +10,7 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += "v86d"
 MACHINE_EXTRA_RRECOMMENDS = "kernel-module-snd-ens1370 kernel-module-snd-rawmidi"
 WKS_FILE ?= "qemux86-grub-efi.wks"
 do_image_wic[depends] += "syslinux:do_populate_sysroot syslinux-native:do_populate_sysroot mtools-native:do_populate_sysroot dosfstools-native:do_populate_sysroot"
+
+MACHINE_FEATURES:append = " pcbios efi"
```
Comment (stlachev) [inline meta-leda-bsp/conf/machine/qemux86-64-extra.conf:14]: Do we have put them inside yaml file of the distro (leda-kirkstone.yaml)?

File: meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend [hunk 2]
```diff
@@ -13,13 +13,13 @@
...
 
 FILES:${PN} += " ${sysconfdir}/systemd/network/can0.network"
 
 do_install:append() {
-    install -d ${D}${sysconfdir}/systemd/network
-    install -m 0644 ${WORKDIR}/can0.network ${D}${sysconfdir}/systemd/network
-}
\ No newline at end of file
+    install -D -m 0644 ${WORKDIR}/can0.network ${D}${systemd_unitdir}/network
+}
```
Comment (stlachev) [inline meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend:25]: This mean for now that we will not have CAN

**Review by stlachev — CHANGES_REQUESTED** [review]
Just to have some opinion about the variables that ware moved inside of leda-kirkstone.yaml

File: meta-leda-bsp/conf/machine/qemux86-64-extra.conf (hunk 1 above)
Comment (mikehaller) [reply meta-leda-bsp/conf/machine/qemux86-64-extra.conf:14]: These machine specific configuration previously were in the local.conf of our distro build repository.
They are supposed to be moved to the machine specific configuration of the meta-leda distro layer.

File: meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend (hunk 2 above)
Comment (mikehaller) [reply meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend:25]: I checked the parent recipe and saw that our implementation was using the incorrect paths on the device (the conf folder of systemd)

The default can0 network systemd network unit file is supposed to be in /lib/systemd/network, which is ${systemd_unitdir}

**Pull request merged**